index.html

```diff
@@ -6,11 +6,9 @@
     <link rel="stylesheet" href="src/css/styles.css" type="text/css"/>
     <script src="http://fb.me/react-0.13.1.js"></script>
     <script src="http://fb.me/JSXTransformer-0.13.1.js"></script>
-    <script src="/socket.io/socket.io.js"></script>
-    <script src="https://cdn.appbase.io/scalr/appbase.js"></script>
+    <script src="bower_components/appbase-js/browser/appbase.js"></script>
```

Comment:
bower is always updated first. it fetches directly from the git repository.
CDN has to be manually updated, plus fetching file locally has it's
benefits.

On Sun, Aug 16, 2015 at 1:23 PM, Kaushik Varanasi <notifications@github.com>
wrote:

> In index.html
> <https://github.com/kaushik94/dejaVu/pull/2#discussion_r37148498>:
>
> > @@ -6,11 +6,9 @@
> >      <link rel="stylesheet" href="src/css/styles.css" type="text/css"/>
> >      <script src="http://fb.me/react-0.13.1.js"></script>
> >      <script src="http://fb.me/JSXTransformer-0.13.1.js"></script>
> > -    <script src="/socket.io/socket.io.js"></script>
> > -    <script src="https://cdn.appbase.io/scalr/appbase.js"></script>
> > +    <script src="bower_components/appbase-js/browser/appbase.js"></script>
>
> Wouldn't it be better if we use the url because if there is an update on
> the file it gets reflected here. Likewise it'll fail if there is a bug in
> the API. That way we'll know. what do you think ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaushik94/dejaVu/pull/2/files#r37148498>.
>